src/main/java/seedu/address/logic/commands/FindNameCommand.java

```diff
@@ -11,9 +11,9 @@
  * Finds and lists all persons in address book whose name contains any of the argument keywords.
  * Keyword matching is case insensitive.
  */
-public class FindCommand extends Command {
```

Comment:
Will it be safer if you add a **new** `FindNameCommand` rather than changing the existing command and all the existing test cases?

Reply:
But we're no longer using just the FindCommand. And I already refactored the code to safely change FindCommand to FindNameCommand, so should be fine I guess